vassal-app/src/main/java/VASSAL/counters/BorderOutline.java

```diff
@@ -55,6 +59,47 @@ public BorderOutline(String type, GamePiece p) {
     setInner(p);
   }
 
+  /**
+   * Comparison Modes for property match
+   */
+  public enum LogicalCompareMode {
+    AND("AND"), //NON-NLS
+    OR("OR"),   //NON-NLS
+    XOR("XOR"), //NON-NLS
+    NOR("NOR"); //NON-NLS
+
+    private static final String[] KEYS = { "Editor.AND", "Editor.OR", "Editor.XOR", "Editor.NOR" };
+
+    String symbol;
+
+    LogicalCompareMode(String symbol) {
+      this.symbol = symbol;
+    }
```

Comment:
Shouldn't these two have visibility specified?

Reply:
It let me make "symbol" private, but compiler doesn't seem to want/allow visibility specification for the constructor

Reply:
There is some weird language thing about enums that I always forget... I think you can't set visibility on them.